File: week2/day3/sum.py

# - Write a function called `sum` that sum all the numbers
#   until the given parameter


p = int(input("Tell me the number until you want to summarize the numbers please: "))

def sum(p):
    x = 0
    y = 0
    while x < p:
        x += 1
        y +=x
    print(y)
sum(p)

File: week8/day3/sum.js
'use strict';
// - Write a function called `sum` that sum all the numbers until the given parameter
// - The function should return the result

function sum(number) {
    var summer = 0;
    for (var i = 0; i < (number); i++) {
        summer += i;
    }
    return summer;
}
console.log(sum(5));
